### DISCUSSION TITLE
Renovate is failing to find public NPM packages (and auto-closing PRs as a result)

### DISCUSSION THREAD

**Discussion by sgoudham** [renovatebot/renovate#32976] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

N/A

### Please tell us more about your question or problem

Renovate has begun auto-closing all existing `npm` PRs across the [catppuccin](https://github.com/catppuccin) organisation as it is failing to find public NPM packages.

### Logs (if relevant)

<details><summary>Logs</summary>

We have Renovate setup on over 100 repositories so it's impossible to list all the logs but here are a few examples:

`catppuccin/highlightjs`
```
WARN: Package lookup failures
{
  "warnings": [
    "Failed to look up npm package @catppuccin/palette",
    "Failed to look up npm package @catppuccin/tailwindcss",
    "Failed to look up npm package rimraf",
    "Failed to look up npm package sass",
    "Failed to look up npm package serve",
    "Failed to look up npm package tailwindcss"
  ]
  "files": [
    "package.json"
  ]
}
```

`catppuccin/discord`
```
WARN: Package lookup failures
{
  "warnings": [
    "Failed to look up npm package @catppuccin/highlightjs",
    "Failed to look up npm package @catppuccin/palette",
    "Failed to look up npm package husky",
    "Failed to look up npm package lint-staged",
    "Failed to look up npm package prettier",
    "Failed to look up npm package sass"
  ]
  "files": [
    "package.json"
  ]
}
```

`catppuccin/jetbrains`
```
WARN: Package lookup failures
{
  "warnings": [
    "Failed to look up npm package @catppuccin/palette",
    "Failed to look up npm package handlebars"
  ]
  "files": [
    "generateFlavours/deps.ts"
  ]
}
```

</details>

**Comment by proton-ab** [comment, marked as answer]
NPM registry is conducting scheduled maintenance, though it would seem that something has gone horribly wrong as they promised installation and package discovery to still be operational - https://status.npmjs.org/

Renovate always closes PRs when it fails to resolve dependency. Perhaps it should be more lenient in this regard? Some of the closed PRs might have had discussions ongoing or labels attached that will not be replicated when new PR is opened.

**Reply by sgoudham** [reply]
Apologies for not conducting more research and pointing straight at Renovate, thank you for the information!

Yeah I'd be interested in learning more about the configuration to maybe stop it from auto-closing but I suppose this is quite a rare incident anyways...

**Reply by viceice** [reply]
if it works again in 24 hours, then renovate should reopen the existing PR

**Reply by sgoudham** [reply]
Can confirm that the existing PRs are being re-opened, thank you and sorry for the noise!
